.github/workflows/ci.yml

```
@@ -12,74 +12,95 @@ on:
 
 jobs:
+  label:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v4
+
+      - name: Determine label
+        id: determine_label
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:23: Double quote to prevent globbing and word splitting

(shellcheck)